src/components/home/AllCollections.js

```diff
@@ -0,0 +1,292 @@
+  };
+
+  const GOTO_LIST = [
+    '/allCollections',
+    '/preSchool',
+    '/grade1to8',
+    '/highSchool',
+    '/college',
+  ];
+
+  const [sideBarStatus, setSideBarStatus] = React.useState(true);
+
+  const closeSideBar = () => {
+    setSideBarStatus(false);
+  };
+  const OpenSideBar = () => {
+    setSideBarStatus(true);
+  };
+
+  return (
+    <div className={classes.root}>
+      <CssBaseline />
+      <AppBar position="fixed" className={classes.appBar}>
+        <Header />
+      </AppBar>
+      <Drawer
+        className={classes.drawer}
+        variant="persistent"
+        open={sideBarStatus}
+        anchor="left"
+        classes={{
+          paper: classes.drawerPaper,
+        }}
+      >
+        <div className={classes.toolbar} />
+        <div className={classes.drawerHeader}>
+          <IconButton onClick={closeSideBar}>
+            {theme.direction === 'ltr' ? (
+              <ChevronLeftIcon />
+            ) : (
+              <ChevronRightIcon />
+            )}
+          </IconButton>
+        </div>
+        <Divider />
+        <List className={classes.list}>
+          {['All', 'Pre-School', 'Grade 1-8', 'High School', 'College'].map(
+            (text, index) => (
+              <Link href={GOTO_LIST[index]} className={classes.link}>
+                <ListItem button key={text}>
+                  <ListItemIcon>
+                    <BookmarkIcon />
+                  </ListItemIcon>
+                  <ListItemText primary={text} />
+                </ListItem>
+              </Link>
+            ),
+          )}
+        </List>
+        <Divider />
+        <List className={classes.list}>
+          {['Test Prep', 'Post Grad'].map((text) => (
+            <ListItem button key={text} disabled>
+              <ListItemIcon>
+                <BookmarkIcon />
+              </ListItemIcon>
+              <ListItemText primary={text} />
+            </ListItem>
+          ))}
+        </List>
+        <Divider />
+        <a href="https://graasp.eu">
+          <Button variant="contained" className={classes.button}>
+            Create Your Own
+          </Button>
+        </a>
+      </Drawer>
+      <main
+        className={clsx(classes.content, {
+          [classes.contentShift]: sideBarStatus,
+        })}
+      >
+        <IconButton
+          color="primary"
+          aria-label="open menu"
+          onClick={OpenSideBar}
+        >
+          <MenuOpenIcon />
+        </IconButton>
+        <Typography variant="body1" display="inline">
+          Open Menu
+        </Typography>
+        <Seo
+          title={APP_NAME}
+          description={APP_DESCRIPTION}
+          author={APP_AUTHOR}
+        />
+        <div style={{ marginLeft: 50 }}>
+          <Typography variant="h3" align="center">
+            {t('Graasp Collections Directory')}
+          </Typography>
+          <Search handleSearch={handleSearch} isLoading={isLoading} />
+          {isLoading ? <Loader /> : renderResults()}
```

Comment:
Very probably it will be easier if the search component could contain the results and the handle function as well. At least for now to avoid mixing everything.

Reply:
I will actually remove the search component on this page. On the home page, there is already the search available, so I don't want to put the searching box everywhere ; )